### DISCUSSION TITLE
Forditás (Translation)

### DISCUSSION THREAD

**Discussion by Akdt21** [EDCD/EDDI#2617] [EDDI Standalone Q&A] [answered]
Ki a faszom forditotta ezt a programot? Jelentkezem forditásra mert ez borzató

**Comment by nepomuk16321** [comment]
First of all a friendly ‘Hello’!
Secondly, I would say you'd better write in English here. It's easier for the programmers that way.
And thirdly, there is a translation page for EDDI. You have to register there, then you can translate all the phrases into Hungarian. But be careful, there is no 1 to 1 translation possible. You should adapt the language script to the translation (grammar).
The translations will be adopted with the next update.
Here is the link to the translation page: https://crowdin.com/project/eddi

Először is, egy barátságos „Hello”!
Másodszor, azt mondanám, hogy jobb, ha angolul írsz ide. Így könnyebb a programozóknak.
És harmadszor, van egy fordítási oldal az EDDI-hez. Ott kell regisztrálnod, aztán lefordíthatod az összes mondatot magyarra. De vigyázz, ott nem lehet 1 az 1hez fordítani. A fordításhoz (nyelvtanhoz) kell igazítani a nyelvi scriptet.
A fordítások a következő frissítéssel kerülnek átvételre.
Itt a link a fordítási oldalra: https://crowdin.com/project/eddi

(@richardbuckle @Tkael I hope the hint to the user is also in your interest)

**Comment by Tkael** [comment, marked as answer]
Thank you @nepomuk16321.

@Akdt21 
We would be pleased to have your assistance with the translation. If you would like to assist, please register an account at the translation page then message me so that I can grant permission to your account to assist with the translation. My account user name is `Tkael`.

Örömmel vennénk a segítségét a fordításban. Ha segíteni szeretne, regisztráljon egy fiókot a fordítási oldalon, majd írjon nekem, hogy engedélyt adjak fiókjának a fordításhoz. A fiókom felhasználói neve `Tkael`.

Google Fordítóval lefordítva. Elnézést kérünk az esetleges nyelvtani hibákért. o7
